addons/full-import/export.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+set -o nounset -o pipefail -o errexit
+
+source /usr/local/pf/addons/full-import/helpers.functions
+source /usr/local/pf/addons/full-import/database.functions
+
+output="$1"
+
+if [ -z "$output" ]; then
+  echo "Missing output parameter"
+  echo "Usage export.sh /path/to/export.tgz [--force]"
```

Comment:
I don't think the `--force` argument is handle in `export.sh`

Reply:
Fixed in 78684d0a6a4e8b0577945bc67529734ff64a8aee

Reply:
If you run `export.sh filename` with `filename` a name of a file (without path), it works but I think `filename` is removed at end of process. If somebody used `export.sh --force /path/to/export.tgz`, you got this issue.

Reply:
Also when using correctly, the `--force` flag doesn't seem to operate as expected because export script continue to copy files from `/root/backup` in place of doing new backups.

Reply:
The `--force` flag actually accepts backups older than 24 hours for cases where your current PF install is broken

In the instructions, we'll state that you must take a backup before running this tool

Reply:
Sorry I misunderstood usage of `--force`